rbx_dom_lua/src/database.json

```diff
@@ -394,7 +420,7 @@
           "BinaryString": ""
         },
         "CancelAirMomentum": {
-          "Bool": true
+          "Bool": false
```

Comment:
Huh, this is kinda weird. I wonder what's up with this diff!

Reply:
It'll break everything randomly and we don't even know it yet.